src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -224,6 +224,8 @@ private function getOrderCustomer(Order $order): ?OrderCustomerForViewing
         $customerStats = $customer->getStats();
         $totalSpentSinceRegistration = Tools::convertPrice($customerStats['total_orders'], $order->id_currency);
 
+        $isB2BEnabled = (bool) Configuration::get('PS_B2B_ENABLE');
```

Comment:
hmm why we didn't use `ConfigurationAdapter` in this class? 🤔

Reply:
Correct. I make the change.

Reply:
Done